src/utils/clack-utils.ts

```diff
@@ -190,7 +190,7 @@ export async function askForWizardLogin(options: {
   const loginSpinner = clack.spinner();
 
   loginSpinner.start(
-    'Waiting for you to click the link above 👆. Take your time.',
+    'Waiting for you to log in using the link above. Once you have logged in, return to this wizard.',
```

Comment:
"have logged in" sounds fishy to me 🤔 

```suggestion
    'Waiting for you to log in using the link above. Once you're logged in, return to this wizard.',
```